#! /bin/sh /usr/share/dpatch/dpatch-run
## 17_texdoc.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: texdoc should follow symlinks, and man pages, and sec fixes, from tetex

@DPATCH@
 texk/texlive/linked_scripts/texdoc/texdoc.tlu |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: texlive-bin-2009~svn13478/texk/texlive/linked_scripts/texdoc/texdoc.tlu
===================================================================
--- texlive-bin-2009~svn13478.orig/texk/texlive/linked_scripts/texdoc/texdoc.tlu	2009-05-26 11:00:08.000000000 +0200
+++ texlive-bin-2009~svn13478/texk/texlive/linked_scripts/texdoc/texdoc.tlu	2009-05-27 22:20:00.000000000 +0200
@@ -6,6 +6,9 @@
     Origial shell script by Thomas Esser, David Aspinall, and Simon Wilkinson.
     Public domain.]]
 --[[ Changelog
+  0.4-debian 2008-07
+  - set support_zipped to true [fk]
+
   0.4 2008-07
   - moved configuration from texmf.cnf to texdoc.cnf
   - added an 'alias' feature
@@ -77,7 +80,7 @@
 -- everything  works for them (look for support_zipped in the code).
 -- If you use this feature, please let us know: if nobody uses it,
 -- we'll drop it at some point.
-support_zipped = false
+support_zipped = true
 
 
 -- BEGIN function definitions (till the 'END' mark)
